doc/modules/classes.rst

```diff
@@ -1218,7 +1218,7 @@ Model validation
    :no-members:
    :no-inherited-members:
 
-**User guide:** See the :ref:`multiclass` section for further details.
+**User guide:** See the :ref:`multiclass_classification` section for further details.
```

Comment:
Would the ref to 'multilabel_classification' be better here?

Reply:
From what I understood, multilabel in `sklearn.multiclass` is a relic from older sklearn versions. And, that we wanted to encourage users to use `sklearn.multioutput` for multilabel instead.

I think the confusion here is the section titles... `multiclass.py` shouldn't have "Multilabel classification" in the title anymore, since that is now handled by `multioutput.py`. 

I've made a change in 2356c8ef206b56d995f52b8b676c3315e513dbbc. But, it's possible I'm off-base here with my understanding.

Reply:
See also https://github.com/scikit-learn/scikit-learn/pull/17811#issuecomment-656940275 for the suggestion that `multilabel` should be associated with `sklearn.multioutput`.

Reply:
Hmm good point. We do talk about multi-label under the 'Multiclass classification strategies' section though:

> This method returns probabilities of class membership in both the single label and multilabel case. Note that in the multilabel case, probabilities are the marginal probability that a given sample falls in the given class. As such, in the multilabel case the sum of these probabilities over all possible labels for a given sample will not sum to unity, as they do in the single label case.

Reply:
> OvR seems to support multilabel, though the other 2 don't.

You're right about that. 

There is related discussion in this issue https://github.com/scikit-learn/scikit-learn/issues/9602 (see https://github.com/scikit-learn/scikit-learn/issues/9602#issuecomment-323948432 in particular) that floats the idea of discouraging and/or deprecating OvR for multilabel. 

Personally, I like the idea of leaving the functionality in OvR for backwards compatibility but using documentation to nudge users towards `sklearn.multioutput` for their multilabel needs. But, that is a bigger discussion and I am a small contributor.

Reply:
Makes sense! In that case leave it as is!

Reply:
It might be good to hear from @jnothman as well considering I've quoted an older comment of his and it might not reflect his current feelings. 

e.g. If we really want to more clearly discourage `sklearn.multiclass.OneVsRestClassifier` for multilabel, there could be additional edits made to make it more explicit that `sklearn.multioutput` is the way to go instead.

Reply:
I don't think it's this PR's job to remove multilabel support from OvR... Multilabel support in OvR is used internally to make binary estimators support multilabel automagically. It's a bit of a hack.

I still think we should be encouraging MultiOutputClassifier for multilabel cases, since OvR is not actually doing one-vs-rest in the multilabel case. MultiOutputClassifier also has a different probability API.

I think it would be good to discuss this with the core dev team and resolve if it is an issue worth breaking people's code for.

Reply:
> I don't think it's this PR's job to remove multilabel support from OvR.

My apologies, I wasn't clear about what I was asking. I was just curious re: your thoughts on documentation _("What should we be encouraging?")_ and not about code. So, what you write here:

> I still think we should be encouraging MultiOutputClassifier for multilabel cases

Is the part that clarifies things for me. Thank you. :)